automatic/vlc/vlc.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>VLC</id>
+    <title>VLC media player</title>
+    <version>2.2.4</version>
+    <authors>VideoLAN Organization</authors>
+    <owners>Rob Reynolds</owners>
```

Comment:
Agreed.  If once you are both happy with this, if you assign to me, I can get the chocolatey user added to that package.